docs/docsite/directive_desc.yml

```diff
@@ -0,0 +1,23 @@
+become: Boolean that controls if privilege escalation is used or not on Task execution.
+become_flags: A string of flag(s) to pass to the privilege escalation program when ``become`` is True.
+become_method: Which method of privilege escalation to use. i.e. sudo/su/etc.
+become_user: User that you 'become' after using privilege escalation, the remote/login user must have permissions to become this user.
+check_mode: A boolean that controls if a task is executed in 'check' mode
+connection: Allows you to change the connection plugin used for tasks to execute on the target.
+environment: A dictionary that gets converted into environment vars to be provided for the task upon execution.
+gather_facts: A boolean that controls if the play will automatically run the 'setup' task to gather facts for the hosts.
+hosts: A list of groups, hosts or host pattern that translates into a list of hosts that are the play's target.
+ignore_errors: Boolean that allows you to ignore task failures and continue with play. It does not affect connection errors.
```

Comment:
s/It does not affect connection errors./Connection errors will always cause an error and stop the playbook run for the  machine./


Not sure what the best wording for "stop the playbook run for the  machine", feel free to change